include/api.php

```diff
@@ -1904,14 +1905,14 @@ function api_favorites_create_destroy($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.